src/acceptance-test/java/net/consensys/orion/acceptance/send/receive/privacyGroup/DualNodesPrivacyGroupsTest.java

```diff
@@ -197,6 +194,38 @@ void createAndFind() {
         firstNodePrivacyGroups[0].getPrivacyGroupId());
   }
 
+  @Test
+  void createAndRetrieve() {
+    final EthClientStub firstNode = NodeUtils.client(firstOrionLauncher.clientPort(), firstHttpClient);
+    final EthClientStub secondNode = NodeUtils.client(secondOrionLauncher.clientPort(), secondHttpClient);
+
+    final String name = "testName";
+    final String description = "testDescription";
+    final String[] addresses = new String[] {PK_1_B_64, PK_2_B_64};
+    // create a privacy group
+    final PrivacyGroup privacyGroup = createPrivacyGroupTransaction(firstNode, addresses, PK_1_B_64, name, description);
+
+    final String privacyGroupId = privacyGroup.getPrivacyGroupId();
+    assertEquals(privacyGroup.getName(), name);
+    assertEquals(privacyGroup.getDescription(), description);
+
+    // get the created privacy group in first node
+    final PrivacyGroup firstNodePrivacyGroup = retrievePrivacyGroupTransaction(firstNode, privacyGroupId);
+
+    assertEquals(firstNodePrivacyGroup.getPrivacyGroupId(), privacyGroupId);
+    assertEquals(firstNodePrivacyGroup.getDescription(), description);
+    assertEquals(firstNodePrivacyGroup.getName(), name);
+    assertArrayEquals(firstNodePrivacyGroup.getMembers(), addresses);
+
+    // get the created privacy group in second node
+    await().atMost(20, TimeUnit.SECONDS).until(
+        () -> retrievePrivacyGroupTransaction(secondNode, privacyGroupId).getPrivacyGroupId().equals(privacyGroupId));
+    final PrivacyGroup secondNodePrivacyGroup = retrievePrivacyGroupTransaction(secondNode, privacyGroupId);
+    assertEquals(secondNodePrivacyGroup.getDescription(), description);
```

Comment:
nit: can you do a dodgy and do an assertThat(secondNodePrivacyGroup).equalsFieldByField(firstNodePrivacyGroup)?

Reply:
done